referencing 7 days as the default (#102)
tlswg/tls-subcerts: draft-ietf-tls-subcerts.md

: Time, in seconds relative to the delegation certificate's
  notBefore value, after which the delegated credential is no longer valid.
  Endpoints will reject delegated credentials that expire more than 7 days

>   from the current time (as described in {{client-and-server-behavior}})
>   based on the default (see {{solution-overview}}.

Author: I think the answers the mail
```suggestion
from the current time {{client-and-server-behavior}} unless a different value is specified in an application profile standard ({{solution-overview}}).
```

## Privacy Considerations

> Delegated credentials can be valid for 7 days (by default) and it is much easier for a

Author: ```suggestion
Delegated credentials can be valid for 7 days (in the absence of an application protocol) and it is much easier for a
```

certificate, a hypothetical signature forgery attack would allow forging a
signature over a delegated credential.
The forged delegated credential could then be used by the attacker as the equivalent of a

> man-in-the-middle certificate, valid for a maximum of 7 days (if the default
> valid_time is used).

Author: ```suggestion
man-in-the-middle certificate, valid for a maximum of 7 days (in the absence of an application protocol).
```
